### DISCUSSION TITLE
Actually makes the button "create codespace" work on empty repo

### DISCUSSION THREAD

**Discussion by MoonLGH** [community/community#54484] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/66781351/236340818-4a36d69a-76a9-468a-b7df-864f24b0b21c.png)
I recently came across a button that wasn't working on a new repository. After some investigation, I realized that the create codespace button was not properly configured to function as intended on a new and empty repository, it will refuses to make a codespace instances on an empty repository

Since this button is an important feature for the repository, I took the initiative to create this issue

I believe it would be beneficial for the repository and its users if this button could be fixed. 

Could someone from the development team take a look at this issue and probably fix it ??
Thank you.

**Comment by Fengyi-Li** [comment, marked as answer]
Hello from the Codespace team, thanks for reporting the issue! We will start to investigate.
